FinalXmL/FinalXmL/Countrydata.cs

```diff
@@ -21,7 +21,7 @@ namespace QuickType
 
 
 
-    public partial class Welcome
+    public partial class Country
     {
```

Comment:
Changed this to country rather than welcome

Reply:
Good... Country is more self-documenting than Welcome is.